pubnative-library/src/net/pubnative/library/vast/VastAd.java

```diff
@@ -23,7 +23,6 @@
 
 import java.util.ArrayList;
 
-import net.pubnative.library.vast.VastAd.Creative.TrackingEvent;
```

Comment:
This just won't compile for me as it was written. I need to remove the import and then refer to the class through its containing class. I didn't OOKLA_EDIT this because I'm thinking of submitting the commit as a PR to the original repository.